+nosnoc/+model/Base.m

```diff
@@ -408,12 +408,12 @@ function verify_and_backfill(obj, opts)
 
                 % some custom handling for objective functions:
                 if strcmp(name, 'f_q')
-                    if obj.f_q == 0
+                    if class(obj.f_q) ~= "casadi.SX" && obj.f_q == 0
```

Comment:
?

Reply:
Comparing a double to and SX even if both are 0 does not work.  Though this is also subtly broken so I made the check different. This is so empty `f_q` `f_q_T` do not get displayed (as was requested when this feature was implemented).